packages/calcite-components/src/components/sheet/sheet.tsx

```diff
@@ -89,19 +89,23 @@ export class Sheet implements OpenCloseComponent, FocusTrapComponent, LoadableCo
   @Prop({ mutable: true, reflect: true }) open = false;
 
   @Watch("open")
-  async toggleSheet(value: boolean): Promise<void> {
+  toggleSheet(value: boolean): void {
     if (this.ignoreOpenChange) {
       return;
     }
 
-    onToggleOpenCloseComponent(this);
     if (value) {
       this.openSheet();
     } else {
       this.closeSheet();
     }
   }
 
+  @Watch("opened")
```

Comment:
@Elijbet This component will also have to be updated if we make any changes to the `OpenCloseComponent` interface mentioned [here](https://github.com/Esri/calcite-design-system/pull/7680#discussion_r1316496111).